### PR TITLE
docs: alokai metadata

### PR DESCRIPTION
**Pull request by Fifciu**
[TFAL-95](https://alokai.atlassian.net/browse/TFAL-95)

[TFAL-95]: https://alokai.atlassian.net/browse/TFAL-95?atlOrigin=eyJpIjoiNWRkNTljNzYxNjVmNDY3MDlhMDU5Y2ZhYzA5YTRkZjUiLCJwIjoiZ2l0aHViLWNvbS1KU1cifQ

### CODE REVIEW

File: docs/content/3.middleware/2.guides/10.logging.md [hunk 1]
```diff
@@ -6,31 +6,64 @@ The middleware application provides a logger instance that automatically attache
...
 }
 ```
 
-In this section, we will explore how to access and use the logger in different parts of the application.
+The provided logger has a second optional argument that allows you to include custom metadata to be attached to the JSON output. For example:
 
-The below examples will demonstrate how to use the application logger in a way that ensures that correct metadata is conveyed in the logs. To achieve that we will use `getLogger` function that will extract the logger instance from the passed.
+```ts
+logger.info("Something went wrong!", {
+  foo: "bar"
```
Comment (bartoszherba) [inline docs/content/3.middleware/2.guides/10.logging.md:44]: do not use such placeholders, add something meaningful

File: docs/content/3.middleware/2.guides/10.logging.md [hunk 2]
```diff
@@ -6,31 +6,64 @@ The middleware application provides a logger instance that automatically attache
 
 The middleware application provides access to the logger in various parts of the system, such as in extensions and integrations. This flexibility allows you to log important events, errors, and other data throughout the lifecycle of your application.
 
-```json[Example log]
+In this section, we will explore how to access and use the logger in different parts of the application.
+
+The below examples will demonstrate how to use the application logger in a way that ensures that correct metadata is conveyed in the logs. To achieve that we will use `getLogger` function that will extract the logger instance from the passed.
```
Comment (dawid-ziobro) [inline docs/content/3.middleware/2.guides/10.logging.md:11]: From the passed... what? :)

Comment (Fifciu) [inline docs/content/3.middleware/2.guides/10.logging.md:11]:
```suggestion
The below examples will demonstrate how to use the application logger in a way that ensures that correct metadata is conveyed in the logs. To achieve that we will use `getLogger` function that will extract the logger instance from the passed argument.
```

File: docs/content/3.middleware/2.guides/10.logging.md [hunk 3]
```diff
@@ -6,31 +6,64 @@ The middleware application provides a logger instance that automatically attache
...
 }
 ```
 
-In this section, we will explore how to access and use the logger in different parts of the application.
+The provided logger has a second optional argument that allows you to include custom metadata to be attached to the JSON output. For example:
 
-The below examples will demonstrate how to use the application logger in a way that ensures that correct metadata is conveyed in the logs. To achieve that we will use `getLogger` function that will extract the logger instance from the passed.
+```ts
+logger.info("Something went wrong!", {
+  foo: "bar"
```
Comment (Fifciu) [inline docs/content/3.middleware/2.guides/10.logging.md:44]:
```suggestion
logger.info("I've just fetched a user!", {
  userId: "123"
```

File: docs/content/3.middleware/2.guides/10.logging.md [hunk 4]
```diff
@@ -6,31 +6,64 @@ The middleware application provides a logger instance that automatically attache
...
+  "message": "Something went wrong!",
+  "timestamp": "2024-10-22T19:04:18.791Z",
+  "severity": "INFO",
+  "alokai": {
+    "context": "middleware",
+    "scope": {
+      ...
+    }
+  },
+  "foo": "bar"
```
Comment (Fifciu) [inline docs/content/3.middleware/2.guides/10.logging.md:60]:
```suggestion
  "message": "I've just fetched a user!",
  "timestamp": "2024-10-22T19:04:18.791Z",
  "severity": "INFO",
  "alokai": {
    "context": "middleware",
    "scope": {
      ...
    }
  },
  "userId": "123"
```

**Review by dawid-ziobro — APPROVED** [review]

**Review by bartoszherba — APPROVED** [review]

**Review by bartoszherba — APPROVED** [review]

**Pull request merged**